emr/application/controllers/Screeners.php

```diff
@@ -140,58 +140,108 @@ public function added_time() {
       $data['last_name'] = $this->ion_auth->user()->row()->last_name;
       $data['employeeid'] = $this->ion_auth->user()->row()->employeeid;
       $data['email'] = $this->ion_auth->user()->row()->email;
-
```

Comment:
I removed those three lines of code!

Reply:
and did it work as expected?

Reply:
Yes, it was working for me without showing the error

Reply:
yes it did

Reply:
Sounds good!